assets/sass/components/reader-revenue-manager/_googlesitekit-rrm-settings-module.scss

```diff
@@ -42,4 +42,40 @@
 			}
 		}
 	}
+
+	.googlesitekit-rrm-settings-edit {
+
+		.googlesitekit-settings-module__fields-group:not(:last-child) .googlesitekit-publication-onboarding-state-notice {
+			margin-bottom: $grid-gap-phone;
+
+			@media (min-width: $bp-desktop) {
+				margin-bottom: $grid-gap-desktop;
+			}
+		}
+
+		.googlesitekit-rrm-settings-edit__snippet-mode p {
+			color: $c-surfaces-on-surface-variant;
+			font-size: $fs-body-sm;
+			letter-spacing: $ls-xs;
+			line-height: $lh-body-sm;
+			margin: 10px 0 0;
+		}
```

Comment:
We could make these styles common as the product ID select dropdown will have a similar footer. However, I will leave it to you.

Reply:
Good call. I've updated the `Select` component to support a `helperText` prop. Thanks!

Reply:
Nice one 👍